docs-src/go/choose-dev-cluster.md

```diff
@@ -0,0 +1,130 @@
+---
+id: choose-dev-cluster
+title: Choose a development Cluster
```

Comment:
Cloud and self hosted are recommended in this section for production, but this section is about development. We could greatly simplify the whole Project setup section by moving Cloud/self-hosted options to a production section. (since it also shows up under "how to run a cloud worker" and CLI sections)